views/render-footer-tracking-events.php

```diff
@@ -0,0 +1,10 @@
+<?php if ($this->has_events_in_cookie): ?>
+    <script type="text/javascript">
+        document.addEventListener("DOMContentLoaded",function() {
+            const url         = "<?php echo add_query_arg('metrilo_clear', 1); ?>";
+            const httpRequest = new XMLHttpRequest();
+            httpRequest.open('GET', url);
+            httpRequest.send();
+        });
+    </script>
+<?php endif; ?>
```

Comment:
So this makes a request to the same page, just for us to clear some cookies. I think we can make it in javascript right away. This seems like an overkill.

Reply:
This one can't be done, because we use cookies that are named after specific wordpress hashes with `COOKIEHASH`.